js/samples/04.ai-apps/h.datasource-azureOpenAI/env/.env.local

```diff
@@ -4,5 +4,4 @@ TEAMSFX_ENV=local
 APP_NAME_SUFFIX=local
 TEAMS_APP_ID=
 TEAMS_APP_TENANT_ID=
-BOT_ID=
-AZURE_OPENAI_ENDPOINT=
```

Comment:
is `AZURE_OPENAI_ENDPOINT` being removed? I still see references to it in the code

Reply:
Its not removed, but it already exists in the sample.env which the instructions point you to so its not needed here.